assets/js/blocks/mini-cart/block.tsx

```diff
@@ -47,6 +47,7 @@ interface Props {
 	colorClassNames?: string;
 	style?: Record< string, Record< string, string > >;
 	contents: string;
+	miniCartIcon?: 'cart' | 'bag' | 'bag-alt';
```

Comment:
We are using `'cart' | 'bag' | 'bag-alt'` in multiple places. Maybe we can consider creating a type for this?
```
type IconType = 'cart' | 'bag' | 'bag-alt';
```

What do you think?